addon_imps/citations/zotero_org.py

```diff
@@ -0,0 +1,5 @@
+from addon_toolkit.interfaces.storage import StorageAddonImp
+
+
+class ZoteroOrgCitationImp(StorageAddonImp):
```

Comment:
this is a crutch because we still don't have citation support inside `gravyvalet`. I tried constructing dummy `CitationAddonImp`, but this has created more problems than it solved